jupyter_sphinx/ast.py

```diff
@@ -127,20 +129,24 @@ def run(self):
         else:
             hl_lines = []
 
-        return [
-            JupyterCellNode(
-                "",
-                docutils.nodes.literal_block(text="\n".join(content)),
-                hide_code=("hide-code" in self.options),
-                hide_output=("hide-output" in self.options),
-                code_below=("code-below" in self.options),
-                linenos=("linenos" in self.options),
-                linenostart=(self.options.get("lineno-start")),
-                emphasize_lines=hl_lines,
-                raises=self.options.get("raises"),
-                stderr=("stderr" in self.options),
-            )
-        ]
+        # A top-level placeholder for our cell
+        cell_node = JupyterCellNode(
+            hide_code=("hide-code" in self.options),
+            hide_output=("hide-output" in self.options),
+            code_below=("code-below" in self.options),
+            linenos=("linenos" in self.options),
+            linenostart=(self.options.get("lineno-start")),
+            emphasize_lines=hl_lines,
+            raises=self.options.get("raises"),
+            stderr=("stderr" in self.options),
+            classes=["jupyter_cell"],
```

Comment:
This class is introduced by the PR, right? That seems like it would warrant documenting [here](https://jupyter-sphinx.readthedocs.io/en/latest/#styling-options).